tests/Integration/inc/classes/subscriber/third-party/plugins/Smush_Subscriber/maybeDeactivateRocketLazyload.php

```diff
@@ -0,0 +1,148 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\classes\third_party\plugins\Smush_Subscriber;
+
+use Brain\Monkey\Functions;
+
+/**
+ * @covers \WP_Rocket\Subscriber\Third_Party\Plugins\Smush_Subscriber::maybe_deactivate_rocket_lazyload
+ * @group ThirdParty
+ * @group Smush
+ * @group WithSmush
+ */
+class Test_MaybeDeactivateRocketLazyload extends SmushSubscriberTestCase {
+	private $option_hook_prefix = 'pre_get_rocket_option_';
+	private $rocket_settings;
+
+	public function setUp() {
+		parent::setUp();
+
+		remove_all_filters( $this->option_hook_prefix . 'lazyload' );
+		remove_all_filters( $this->option_hook_prefix . 'lazyload_iframes' );
+		remove_all_filters( 'update_option_wp_rocket_settings' );
+
+		$this->rocket_settings = get_option( 'wp_rocket_settings', [] );
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+
+		update_option( 'wp_rocket_settings', $this->rocket_settings );
+
+		$this->rocket_settings = null;
+	}
+
+	public function testShouldNotDisableWPRocketLazyLoad() {
+		Functions\expect( 'update_rocket_option' )->never();
+
+		// Smush not enabled, WPR enabled.
+		$this->setSmushSettings(
+			false,
+			[]
+		);
+
+		$this->setRocketSettings(
+			[
+				'lazyload'         => 1,
+				'lazyload_iframes' => 1,
+			 ]
+		);
+
+		$this->subscriber->maybe_deactivate_rocket_lazyload();
+
+		$this->assertFalse( empty( get_rocket_option( 'lazyload' ) ) );//
+		$this->assertFalse( empty( get_rocket_option( 'lazyload_iframes' ) ) );
```

Comment:
- Use [`assertNotEmpty`](https://phpunit.readthedocs.io/en/7.3/assertions.html#assertempty) here instead.
- Instead of using `get_rocket_option`, create a helper method and pull from the database. Why? What if for some reason `get_rocket_option` is not working as expected. By pulling directly from the database we can ensure the state is what we expected for this test.